docker/spicedb/schema.zed

```diff
@@ -16,7 +16,7 @@ definition thing {
 	// These permission are made for only list purpose. It helps to list users have only particular permission excluding other higher and lower permission.
 	permission admin_only = admin
 	permission edit_only = edit - admin
-	permission viewer_only = view - edit
+	permission contributor_only = view - edit
 
```

Comment:
Since `view` and `edit` are permission and they are not relations, Lets change this to `view_only` 
```suggestion
	permission view_only = view - edit
```